Docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_ROOT_PASSWORD: rootpassword
+      MYSQL_DATABASE: todolist
+    volumes:
+      - mysql_data:/var/lib/mysql
+    networks:
+      - todolist_network
+
+  app:
+    build: .
+    depends_on:
+      - db
+    environment:
+      - DATABASE_URL=mysql://root:rootpassword@db/todolist
```

Comment:
The DATABASE_URL environment variable contains the root password in plain text. This is not secure. Consider using environment variables or a secrets management tool to handle sensitive information.